src/swarm/neo/request/RequestEventDispatcher.d

```diff
@@ -251,9 +251,19 @@ public struct RequestEventDispatcher
 
         /// Event being awaited.
         EventRegistration event;
+
+        /// Is this instance currently enabled? If false, it will be ignored
+        /// when dispatching events.
+        bool enabled;
```

Comment:
Could you use `event.active.none` (i.e. `!event.active`) to indicate that this instance is disabled?

Reply:
Unfortunately not, as the loops in `register` and `unregister` match the fiber/event pair.